docs/help-center/tim-features/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: /tim-features
+title: TIM Features
+hide_title: true
+---
+
+import DocCardList from '@theme/DocCardList';
+
+<div class="hero hero--primary">
+  <div class="container">
+    <h2 class="hero__title">TIM Features</h2>
+  </div>
+  
+</div>
+<br></br>
+
+:::note
+
+You will only have access to TIM Features if your account is on a TIM server. Probably you will not see these features. 
```

Comment:
This is weird. We aim to onboard a lot of customers to TIM, so probably just works for the first period of time. I would remove this sentence, so the content is suitable for months to come. It suffices to say that you only see TIM Features if you are on a TIM registered server.